samples.http

```diff
@@ -28,7 +28,7 @@ Content-Type: application/json
 
 
 ### Add highlight to existing match
-POST {{baseUrl}}/matches/1/highlights HTTP/1.1
+POST {{baseUrl}}/matches/59156/highlights HTTP/1.1
```

Comment:
This example won't work, because it's not a valid ObjectId.
In addition, we might have different databases and the id only exists in your environment.
The REST client might have a solution for this in the extension documentation.